pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/fwd/MultiValueVarByteRawIndexCreator.java

```diff
@@ -119,4 +117,55 @@ public void close()
       throws IOException {
     _indexWriter.close();
   }
+
+  /**
+   * The actual content in an MV array is prepended with 2 prefixes:
+   * 1. elementLengthStoragePrefix - bytes required to store the length of each array element
+   * 2. numberOfElementsStoragePrefix - Number of elements in the array
+   *
+   * This function returns the total bytes needed to store (1) elementLengthStoragePrefix
+   */
+  public static int getElementLengthStoragePrefix(int maxNumberOfElements) {
```

Comment:
nit: considering that this function returns a length in bytes, can you rename the function to indicate this? e.g. `getElementLengthStoragePrefixInBytes`
Also, this accounts for the worst case size of each (i.e. max). Perhaps the function description and name should also indicate this.